tests/AdaptiveExpressions.Tests/ExpressionParserTests.cs

```diff
@@ -1196,9 +1196,11 @@ public class ExpressionParserTests
             #endregion
         };
 
+        public static CultureInfo EnUsCultureInfo => new ("en-US");
+
         public static IEnumerable<object[]> DataForThreadLocale => new[]
         {
-            Test("replace(addDays(timestamp, 1, '', 'en-US'), '20', '')", "3/16/18 1:00:00 PM"),
+            Test("replace(addDays(timestamp, 1, '', 'en-US'), '20', '')", new DateTime(2018, 3, 16, 13, 0, 0).ToString(string.Empty, EnUsCultureInfo).Replace("20", string.Empty)),
```

Comment:
I changed this to be en-us culture explicitly (to ignore user settings) because this test was failing on my machine where my date format is YYYY-MM-DD.